nep-0000-template.md

```diff
@@ -1,83 +1,118 @@
 ---
 NEP: 0
-Title: NEP Purpose and Guidelines
-Author: Todd Codrington III <satoshi@fakenews.org>
+Title: NEP Template
+Authors: Todd Codrington III <satoshi@fakenews.org>
+Status: Approved
 DiscussionsTo: https://github.com/nearprotocol/neps/pull/0000
-Status: Draft
-Type: Standards Track
-Category: Contract
-Created: 03-Mar-2022
+Type: Developer Tools
+Version: 1.1.0
+Created: 2022-03-03
+Last Updated: 2023-03-07
```

Comment:
Just to be consistent, let's avoid spaces in the field names:

```suggestion
LastUpdated: 2023-03-07
```

We might reconsider this, but currently, it is the only place that is misaligned, so I vote for fixing it here for now and then iterating on it later.